homework.txt

```diff
@@ -0,0 +1,41 @@
+Create a calculator 
+
+```
+var num1;
+var num2;
+function calculator (num1, num2, operation) {
+if (operation === 'add' || operation === 'subtract' ||
+ operation === 'multiply' || operation=== 'divide'){
+  var result='';
+  if (operator === 'add') {
+    result = num1 + num2
+  } else if (operator === 'subtract') {
+    result = num1 - num2
+  } else if (operator === 'multiply') {
+    result = num1 * num2
+  } else if (operator === 'divide') {
+    result = num1 / num2
+  }
+  return result;
+   }
+}
```

Comment:
Correct syntax of using `||`, however there's no need for them here as doing 
```js
function calculator (num1, num2, operation) {
  var result='';
  if (operator === 'add') {
    result = num1 + num2
  } else if (operator === 'subtract') {
    result = num1 - num2
  } else if (operator === 'multiply') {
    result = num1 * num2
  } else if (operator === 'divide') {
    result = num1 / num2
  }
  return result;
}
```

Also, for a calculator, we'd usually want to get a number back not a string, because we might need to use the returned calculated value in other operations. To fix this, we can `return num1 + num2` for example to give us back a number, not a string. Let me know if you'd like more information about this.

Reply:
Yes, please I like to get more information 
it's in somewhere it's not too clear